apps/web/components/I18nLanguageHandler.tsx

```diff
@@ -2,26 +2,28 @@ import { useTranslation } from "next-i18next";
 import { useEffect } from "react";
 
 import { trpc } from "@calcom/trpc/react";
+import { CalComVersion } from "@calcom/ui/components/credits/Credits";
```

Comment:
Ideally this goes into a different file than its current component. Then, it can be reused across both components that need it.